alpine-model/src/main/java/alpine/model/Permission.java

```diff
@@ -85,6 +85,11 @@ public class Permission implements Serializable {
     @JsonIgnore
     private List<ManagedUser> managedUsers;
 
+    @Persistent(mappedBy = "permissions")
+    @Order(extensions = @Extension(vendorName = "datanucleus", key = "list-ordering", value = "username ASC"))
```

Comment:
The `@Order` part can be removed. I don't think the order of keys is important. Also, `ApiKey` has no `username` field.